### ISSUE TITLE
依赖包和版本问题

### ISSUE THREAD

**Issue by Ostarsion** [modelscope/modelscope#949]
如果用pip安装modelscope，是需要在requirements文件夹下依次挑选需要的依赖文件说明来安装吗，毕竟requirements.txt指向的hug.txt只包含了三个拓展包，此外pytorch是只支持1.7以下的版本吗

**Comment by slin000111** [comment]
> 看懂了，是还需要执行对应领域的pip指令来安装对应依赖是吗

是的。比如，体验NLP领域模型，`pip install "modelscope[nlp]" -f https://modelscope.oss-cn-beijing.aliyuncs.com/releases/repo.html`
